homeassistant/components/script/__init__.py

```diff
@@ -237,7 +237,7 @@ async def _async_process_config(hass, config, component) -> bool:
     for config_key in extract_domain_configs(config, DOMAIN):
         conf: dict[str, dict[str, Any] | BlueprintInputs] = config[config_key]
 
-        for object_id, config_block in conf.items():
+        for key, config_block in conf.items():
```

Comment:
`object_id` renamed to `key` throughout the module to not confuse the key used in YAML with the 2nd half of the `entity_id`.